byway/src/main/java/com/google/sps/data/Recommendation.java

```diff
@@ -23,17 +23,20 @@
   /** We may need to add more fields later, so making a class seemed reasonable. */
   private final long id;
   private final String placename; 
+ 
   public static final String KIND = "Recommendation";
   
   public Recommendation(long id, String placename) {
     this.id = id;
     this.placename = placename;
+   
   }
 
   public static Recommendation fromEntity(Entity entity){
     checkArgument(entity.getKind().equals(KIND), "Wrong Entity kind, expected %s, got %s", KIND, entity.getKind());
     long id = entity.getKey().getId();
     String placename = (String) entity.getProperty("placename");
+
     return new Recommendation(id, placename);
```

Comment:
Thought you deleted this class? Do you need to merge master to pick up that change?

Reply:
After checkout I deleted Recommendation.java and Stop.java since they are both in master.